src/Pages/ItemDetail/Components/ProductInfo/Delivery.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from "react";
+import "./Delivery.scss";
+class Delivery extends Component {
+  constructor() {
+    super();
+    this.state = {
+      height: 72.6,
+      refundHeight: 72.6,
+      reviewHeight: 72.6,
+      isDeliveryShown: false,
+      isRefundShown: false,
+      isReviewShown: false,
+      category: "",
+      price: "",
+      enviromentFriendly: false,
+      differentColor: [],
+      size: [],
+    };
+  }
+
+  deliveryToggle = () => {
+    const { isDeliveryShown } = this.state;
+    isDeliveryShown
+      ? this.setState({
+          height: 72.6,
+          isDeliveryShown: false,
+        })
+      : this.setState({
+          height: "auto",
+          isDeliveryShown: true,
+        });
+  };
+
+  render() {
+    const { height, isDeliveryShown } = this.state;
+    return (
+      <div className="Delivery" style={{ height }}>
+        <div className="container">
+          <div className="deliveryText">
+            <span>배송</span>
+          </div>
+          <div className="typeNarrow">
+            <div className="deliveryType">
+              <span>일반배송</span>
+            </div>
+            <div className="arrow">
+              <i
+                className={
+                  isDeliveryShown ? "fas fa-angle-up" : "fas fa-angle-down"
+                }
+                onClick={() => {
+                  this.deliveryToggle();
+                }}
```

Comment:
```suggestion
                onClick={() => this.setState({ isDeliveryShown: !isDeliveryShown })}
```

- 한 번만 사용 되는 + 로직이 짧은 경우 굳이 함수를 따로 정의하지 않아도 괜찮습니다.
- 말씀드린대로 같은 state 데이터로 한번에 관리할 수 있는 UI인지 고민해보시기 바랍니다!